base/stats.go

```diff
@@ -461,6 +461,8 @@ type DatabaseStats struct {
 	DocWritesXattrBytes *SgwIntStat `json:"doc_writes_xattr_bytes"`
 	// Highest sequence number seen on the caching DCP feed.
 	HighSeqFeed *SgwIntStat `json:"high_seq_feed"`
+	// Number of bytes written over public interface for REST api
+	HTTPBytesWritten *SgwIntStat `json:"http_bytes_written"`
```

Comment:
This seems ambiguous if we're not including Admin or Metric API data in it. Should this be specifically named after the public API?

Reply:
renamed to PublicRestBytesWritten to match NumPublicRestRequests